public/data/nazmulhassan.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "MD NAZMUL HASSAN",

```

```suggestion
    "name": "MD Nazmul Hassan",
```

It looks better when its small, it shouldn't be all caps.

---

What does MD stand for?

---

> What does MD stand for?

Muhammad

---

Then, I suggest it me Muhammad Nazmul Hassan instead of MD

```suggestion
    "name": "Nazmul Hassan Nazmul Hassan",

```

---

> Then, I suggest it me Muhammad Nazmul Hassan instead of MD

Please commit this @nazmul19

---

> Then, I suggest it me Muhammad Nazmul Hassan instead of MD

It's their profile, they can use the name they choose

---

Yes true, deleting the suggestion.